x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/SecurityRestFilter.java

```diff
@@ -93,22 +89,17 @@ public void handleRequest(RestRequest request, RestChannel channel, NodeClient c
             return;
         }
 
-        if (extractClientCertificate) {
-            HttpChannel httpChannel = request.getHttpChannel();
-            SSLEngineUtils.extractClientCertificates(logger, threadContext, httpChannel);
-        }
```

Comment:
The Security plugin now incorporates this when it builds the thread context before running the authentication.